src/commands/diff.ts

```diff
@@ -159,17 +171,31 @@ export default class Diff extends Command {
   }
 
   outputYAML(diffOutput: AsyncAPIDiff, outputType: string) {
-    if (outputType === 'breaking') {
-      this.log(diffOutput.breaking() as string);
-    } else if (outputType === 'non-breaking') {
-      this.log(diffOutput.nonBreaking() as string);
-    } else if (outputType === 'unclassified') {
-      this.log(diffOutput.unclassified() as string);
-    } else if (outputType === 'all') {
-      this.log(diffOutput.getOutput() as string);
-    } else {
-      this.log(`The output type ${outputType} is not supported at the moment.`);
-    }
+    this.log(genericOutput(diffOutput, outputType) as string);
+  }
+
+  outputMarkdown(diffOutput: AsyncAPIDiff, outputType: string) {
+    this.log(genericOutput(diffOutput, outputType) as string);
+  }
+}
+
+/**
+ * A generic output function for diff output
+ * @param diffOutput The diff output data
+ * @param outputType The output format requested by the user
+ * @returns The output(if the format exists) or a message indicating the format doesn't exist
+ */
+function genericOutput(diffOutput: AsyncAPIDiff, outputType: string) {
+  if (outputType === 'breaking') {
```

Comment:
Hey,

Just a minor thing, but do you think it's worth adding a switch statement vs the if conditions? Would make extending easier in the future?